packages/teleport/src/services/apps/apps.ts

```diff
@@ -31,6 +31,8 @@ const service = {
         apps: items.map(makeApp),
         startKey: json?.startKey,
         totalCount: json?.totalCount,
+        paginationUnsupported:
+          json?.startKey === null && json?.totalCount === null,
```

Comment:
I left a comment on the backend PR regarding how we detect if pagination is supported or not. In my experience, in most cases accepting only null and not undefined generates more headaches than structuring the code so that both null and undefined result in the same behavior.

Reply:
replied https://github.com/gravitational/teleport/pull/18262#discussion_r1018214854